Library/OcBootManagementLib/ImageLoader.c

```diff
@@ -980,10 +980,9 @@ InternalEfiLoadImage (
                  );
     } else {
       //
-      // We verified the image, but contained garbage, or we are trying to secure boot a Fat slice.
-      // This should not happen.
+      // We verified the image, but contained garbage.
+      // This should not happen, just abort.
```

Comment:
Reword this comment that this can happen when a trusted binary failed to load for some reason.

Reply:
I agree, was just thinking up some wording.